api/services/projects.py

```diff
@@ -5,5 +5,10 @@
 
 
 class ProjectService(AppService):
-    def get_items(self) -> List[Project]:
-        return self.db.query(Project).all() or []
+    def get_items(self, offset, limit, status) -> List[Project]:
+        query = self.db.query(Project)
+        if status == "active":
+            query = query.filter(Project.activation == True)
```

Comment:
I ran into a build failure with checking bools, too, for the templates endpoint. For flake8 to be happy, it expects the following, I think.
`query = query.filter(Project.activation is True)`
Same for line 13.

Reply:
Solved, thank you. I wonder why locally is passing, I'll check later, probably different versions.